content/docs/other/a_word_on_backward_compatibility.md

```diff
@@ -30,9 +33,10 @@ This is the current list of feature flags:
 | 0.53                                   | 0.53      | Will activate register\_cmdb\_top\_scope. And all things 0.51 and down activated. |
 | register\_cmdb\_top\_scope             | 0.53      | Will register all cmdb top scope variables automatically in the templates. |
 | 0.51                                   | 0.51      | Will load Rex::Constants and the CMDB by default. And all things 0.47 and down activated. |
-| no\_autodie                            | 1.0       | Will disable autodie feature. |
-| no\_path\_cleanup                      | 0.44      | Rex cleans the path before executing a command. With this feature Rex doesn't cleanup the path. |
+| disable\_taskname\_warning             | 0.47      | |
```

Comment:
Proposed description:

Disable warning about invalid task names (they should match `/^[a-zA-Z_][a-zA-Z0-9_]*$/`)

Reply:
Ok. Thanks for the proposed description! Added.